modules/virt-enabling-volume-snapshot-boot-source.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-creating-and-using-boot-sources.adoc
+
+:_content-type: PROCEDURE
+[id="virt-enabling-volume-snapshot-boot-source_{context}"]
+= Enabling a volume snapshot as a boot source
+
+You can import a volume snapshot as a boot source by setting the parameter in the `DataImportCron` namespace. Although `DataImportCron` was originally designed to maintain only PVC sources, volume snapshot sources scale better than PVC sources for certain storage types.
+
+.Prerequisites
+
+* You must have access to a volume snapshot with the operating system image.
+
+.Procedure
+
+. Open the virtual machine's (VM) storage profile and review the  `dataImportCronSourceFormat` specification of the `DataImportCron` namespace to confirm whether or not the VM is using PVC or volume snapshot by default.
++
+[source,terminal,subs="attributes+"]
+----
+$ oc edit -n {DataImportCron} storageprofile <storage_class>
```

Comment:
So storage profiles are not namespace scoped, `-n {DataImportCron}` can be dropped